src/main/java/appeng/core/sync/packets/PacketClick.java

```diff
@@ -94,6 +104,7 @@ public PacketClick( final BlockPos pos, final EnumFacing side, final float hitX,
 		data.writeFloat( this.hitY = hitY );
 		data.writeFloat( this.hitZ = hitZ );
 		data.writeByte( hand.ordinal() );
+		data.writeBoolean( this.leftClick = leftClick );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract the assignment out of this expression. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AAssignmentInSubExpressionCheck)